src/main/java/de/tum/cit/aet/artemis/core/web/open/PublicAccountResource.java

```diff
@@ -96,6 +98,7 @@ public PublicAccountResource(AccountService accountService, UserService userServ
      */
     @PostMapping("register")
     @EnforceNothing
+    @LimitRequestsPerMinute(type = RateLimitType.ACCOUNT_MANAGEMENT)
     public ResponseEntity<Void> registerAccount(@Valid @RequestBody ManagedUserVM managedUserVM) throws URISyntaxException {
```

Comment:
Doesn't this mean that also Administrators can now only register 5 users per Minute? Wouldn't registration usually happen in batches there? Or is this handled by another endpoint?